test/integration/rules/frame-title/frame-title.html

```diff
@@ -64,6 +54,16 @@
   role="presentation"
   tabindex="0"
 ></iframe>
+<iframe
+  src="/integration/rules/frame-title/frames/level1a.html"
+  id="violation8"
+  role="presentation"
+></iframe>
+<iframe
+  src="/integration/rules/frame-title/frames/level1a.html"
+  id="violation9"
+  role="none"
```

Comment:
Add in the test from the issue. I'd like to see that working.

```suggestion
  role="none"
  title="   "
```